sphinx/writers/latex.py

```diff
@@ -583,11 +583,18 @@ def visit_title(self, node):
             self.this_is_the_title = 0
             raise nodes.SkipNode
         elif isinstance(parent, nodes.section):
+            short = ''
+            if 'image' in str(node):
+                elementText = []
+                for n in node.children:
+                    if isinstance(n, nodes.Text):
+                        elementText.append(n.astext().strip())
```

Comment:
You might want to use `.translate(tex_escape_map)` on this so you don't introduce special characters (eg. `]`) inadvertently.